python/mujinplanningclient/binpickingplanningclient.py

```diff
@@ -628,17 +628,17 @@ def SetCurrentLayoutDataSendOnObjectUpdateData(self, doUpdate, containername=Non
         taskparameters.update(kwargs)
         return self.ExecuteCommand(taskparameters, fireandforget=fireandforget)
 
-    def ManuallyPlacePackItem(self, packFormationComputationResult=None, inputPartIndex=None, placeLocationNames=None, placedTargetPrefix=None, dynamicGoalsGeneratorParameters=None, orderNumber=None, numLeftToPick=None, timeout=10, fireandforget=False):
+    def ManuallyPlacePackItem(self, packFormationComputationResult=None, inputPartIndex=None, placeLocationNames=None, placedTargetPrefix=None, packFormationParameters=None, orderNumber=None, numLeftToPick=None, timeout=10, fireandforget=False):
         """Places an item according to the pack formation assuming the item is placed manually and updates robotbridge state
 
         Args:
-            packFormationComputationResult:
-            inputPartIndex:
-            placeLocationNames:
-            placedTargetPrefix:
-            dynamicGoalsGeneratorParameters:
-            orderNumber:
-            numLeftToPick:
+            packFormationComputationResult (json): Pack formation that we try to follow.
+            inputPartIndex (int): Index of the item from packFormationComputationResult (1-based)
+            placeLocationNames (list(str)): Place location names of the order request.
+            placedTargetPrefix (str): Name prefix of items placed to the desired place location.
+            packFormationParameters (json): Placement parameters that should be used to generate final placement position of the item.
```

Comment:
This is the top-level schema, right?

Reply:
Discussed offline, the answer is yes. We'll take care of this when the clients are generated.